app/assets/javascripts/pretty-text/addon/emoji.js

```diff
@@ -88,6 +88,19 @@ export function performEmojiUnescape(string, opts) {
       classes += ` ${opts.class}`;
     }
 
+    if (opts.emojiDenyList) {
+      let deniedEmojiList = opts.emojiDenyList.split("|");
+      deniedEmojiList.forEach((emoji) => {
+        if (aliases[emoji]) {
+          aliases[emoji].forEach((alias) => deniedEmojiList.push(alias));
+        }
```

Comment:
Instead of handling the alises here, perhaps a better approach is to have a method on the server that returns the list of denied emoji list which can be used in multiple places?